backend/src/main/java/hu/flowacademy/konyhatunder/model/Image.java

```diff
@@ -0,0 +1,38 @@
+package hu.flowacademy.konyhatunder.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.GenericGenerator;
+
+import javax.persistence.*;
+
+@Data
+@Entity
+@Table(name = "image")
+@AllArgsConstructor
+@NoArgsConstructor
+public class Image {
+    @Id
+    @GeneratedValue(generator = "uuid")
+    @GenericGenerator(name = "uuid", strategy = "uuid2")
+    @JsonIgnore
+    private String id;
+
+    private String fileName;
+
+    private String fileType;
+
+    @Lob
+    @JsonIgnore
+    private byte[] data;
```

Comment:
Itt nem lehetne ignore helyett maskepp formazni az adatot? Esetleg a DTO-ban? Illetve, ha az entity sosem megy ki kozvetlenul, csak dto-va alakitva nincs is szukseg a jsonos annotaciokra itt.

Reply:
ehhez a recept visszaadást is át kellene alakitani ha gondolod külön issura felvesszük

Reply:
Erdemes lehet, nem high prio, de szebb lenne, es eleg lenne a serviceben eldonteni hogy adtok-e kepet vagy sem (pl egy kulon kep nelkuli recept dto jol johet ha valahol csak kep nelkul listaztok (mar ha van ilyesmi tervben)).